firefox-ios/Client/Frontend/TabContentsScripts/FormAutofillHelper/FormAutofillHelper.swift

```diff
@@ -18,6 +18,17 @@ struct AutofillFieldValuePayload {
     let fieldData: Any?
 }
 
+extension WKScriptMessage {
+    func decodeBody<T: Decodable>(as type: T.Type) -> T? {
```

Comment:
nit: Is this is a one off instance?

If its one time usage only in this class then we can move it to this class 
```
    // MARK: Script message decoder
    func decodeBody<T: Decodable>(as type: T.Type, body: Any) -> T? {
        guard
            let dict = (body as? [String: Any]),
            let data = try? JSONSerialization.data(withJSONObject: dict, options: []) else {
            return nil
        }
        return try? JSONDecoder().decode(type, from: data)
    }
 ```
 


 Otherwise if you plan to use it multiple times then best to move it in one of these areas
 
 <img width="248" alt="Screenshot 2024-03-18 at 2 37 11 PM" src="https://github.com/mozilla-mobile/firefox-ios/assets/8919439/88b73c74-808a-4834-b36b-e765e5b9fb5f">
<img width="272" alt="Screenshot 2024-03-18 at 2 37 38 PM" src="https://github.com/mozilla-mobile/firefox-ios/assets/8919439/cc31e335-e3d5-4faf-9cdd-ff3e8b416f35">

Reply:
I can move it, it's also used for cc